test/Confluent.Kafka.IntegrationTests/Tests/AssignPastEnd.cs

```diff
@@ -23,12 +23,12 @@
 
 namespace Confluent.Kafka.IntegrationTests
 {
-    /// <summary>
-    ///     Test functionality of Consumer.Consume when assigned to offest
-    ///     higher than the offset of the last message on a partition.
-    /// </summary>
     public static partial class Tests
     {
+        /// <summary>
+        ///     Test functionality of Consumer.Consume when assigned to offest
+        ///     higher than the offset of the last message on a partition.
+        /// </summary>
```

Comment:
Yes, but if you assign an invalid offset then auto.offset.reset will kick in - it defaults to end, that's why you're not seeing any messages which is expected by the test, but coming in and seeing this test it might seem that it wont consume if you give it an invalid offset, which isnt true. So make it more explicit what is actually going on by setting that config property, or making a comment.